src/app/frontend/persistentvolume/detail/info_component.js

```diff
@@ -41,15 +41,13 @@ export default class PersistentVolumeInfoController {
    * @export
    */
   getPersistentVolumeClaimDetailsHref() {
-    if(this.persistentVolume.claim) {
-      let claim = this.persistentVolume.claim.split('/')
-      if(claim.length >= 2) {
-        let namespace = claim[0]
-        let claimName = claim[1]
-        return this.state_.href(stateName, new StateParams(namespace, claimName));
+    if (this.persistentVolume.claim) {
+      let claim = this.persistentVolume.claim.split('/');
```

Comment:
Can `claim` be null?

Reply:
Well, forgot my comment. It was okay. You can revert because null check should be here :+1: